apps/shipments/geojson.py

```diff
@@ -104,6 +129,9 @@ def as_point_feature(self):
                 "source": self.source,
             })
         except Exception:
+            LOG.error(f'Device tracking as_point_feature exception {Exception}.')
```

Comment:
This isn't going to print out anything but the Exception class. If you want to capture the details of the exception, you have to catch it like this:
`except Exception as ex:`
and then print out `ex`.